README.md

```diff
@@ -34,6 +34,10 @@
 - **UMD compatible**, you can use it with any module loader
 - Built with TypeScript
 
+## ⏭ Next
+
+The [`next` branch](https://github.com/algolia/algoliasearch-client-javascript/tree/next) hosts the output of the new major version (`v5`). This version is generated from the [`API Clients Automation` monorepo](https://github.com/algolia/api-clients-automation/) and is where you can follow the development and make contributions.
```

Comment:
It might not be the best way to phrase it but we should highlight the fact that it's only the output, as we want to centralize contributions on the monorepo

Reply:
then it's `the output of the generation of the new major version`, the new major version doesn't actually output anything, but it get's cumbersome

Reply:
```suggestion
The [`next` branch](https://github.com/algolia/algoliasearch-client-javascript/tree/next) hosts the code of the new major version (`v5`). This version is generated from the [`API Clients Automation` monorepo](https://github.com/algolia/api-clients-automation/) and is where you can follow the development and make contributions.
```

this may be good too, not source